CodeEdit/Features/CodeFile/CodeFile.swift

```diff
@@ -71,6 +72,12 @@ final class CodeFileDocument: NSDocument, ObservableObject, QLPreviewItem {
 
     @Published var cursorPosition = (1, 1)
 
+    /// Publisher for isDocumentEdited property
+    var isDocumentEditedPublisher: AnyPublisher<Bool, Never> {
+        isDocumentEditedSubject.eraseToAnyPublisher()
+    }
+    private let isDocumentEditedSubject = PassthroughSubject<Bool, Never>()
```

Comment:
Here also, maybe define the variable above its usages

Reply:
Done